docs/core/testing/unit-testing-mstest-runner-exit-codes.md

```diff
@@ -26,3 +26,9 @@ The MSTest runner uses known exit codes to communicate test failure or app error
 | `10` | The exit code `10` indicates that the test adapter, Testing.Platform Test Framework, MSTest, NUnit, or xUnit, failed to run tests for an infrastructure reason unrelated to the test's self. An example is failing to create a fixture needed by tests. |
 
 To enable verbose logging and troubleshoot issues, see [MSTest runner extensions: Troubleshoot](unit-testing-mstest-runner-extensions.md#troubleshoot).
+
+## Ignoring specific exit codes
+
+MSTest runner is designed to be strict by default but allows for configurability. As such, it is possible for users to decide that some exit codes should be ignored (an exit code of `0` will be returned instead of the original exit code).
```

Comment:
```suggestion
MSTest runner is designed to be strict by default but allows for configurability. As such, it's possible for users to decide which exit codes should be ignored (an exit code of `0` will be returned instead of the original exit code).
```